gerrymandering/gerrymandering.ipynb

```diff
@@ -81,14 +76,14 @@
     "races     = {'w':'White', 'b':'Black', 'a':'Asian', 'h':'Hispanic', 'o':'Other'}\n",
     "\n",
     "color_points = hv.NdOverlay(\n",
-    "    {races[k]: gv.Points([0,0], crs=crs.PlateCarree()).opts(color=v) for k, v in color_key.items()})"
+    "    {races[k]: gv.Points([0,0], crs=ccrs.PlateCarree()).opts(color=v) for k, v in color_key.items()})"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next, we'll load the 2010 US Census, with the location and race or ethnicity of every US resident as of that year (300 million data points), and define a plot using datashader to show this data with the given color key:"
+    "Next, we'll load the 2010 US Census, with the location and race or ethnicity of every US resident as of that year (300 million data points) and define a plot using datashader to show this data with the given color key. While we would normally use Pandas to load in data, we will use Dask instead to take advantage of `persist()`, which saves all the data points onto the memory. If we were to not use `persist()` then every time we zoom it will have to read from disk."
```

Comment:
```suggestion
    "Next, we'll load the 2010 US Census, with the location and race or ethnicity of every US resident as of that year (300 million data points) and define a plot using datashader to show this data with the given color key. While we would normally use Pandas to load in data, we will use Dask instead for speed since it can use all the available cores on your machine. We also "persist" the data into memory, which will be faster as long as you have enough memory; otherwise every time we zoom it would have to read from disk."
```

(It was previously incorrect; there is no need to use Dask to be able to persist data in memory -- in fact it's the opposite: Pandas _always_ persists the data; only Dask lets you choose whether or not to. It's just that in this case we do want to persist it, and are using Dask for its speed, not for its delayed or dynamic data reading.)

Reply:
Thank you for clarifying this. I'll make those changes now.